README.md

```diff
@@ -1 +1,223 @@
-# Wheels-4-Rent-frontend
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
This was a little confusing because instead of making the buttons not appear for non-admins, you include them albeit in a non-functional state.

Therefore:

- [ ] OPTIONAL: Since your buttons are still visible for non-admin users, I recommend that you add this note to your readme in the "usage" section, for clarification purposes and to prevent confusion:

![image](https://github.com/Log-benjamin/Wheels-4-Rent-frontend/assets/57421823/f1257b8f-dbca-4ba9-a9b3-3dd96bc49bee)

Another way is to create a user and then modify it through the console (I did this):

![image](https://github.com/Log-benjamin/Wheels-4-Rent-frontend/assets/57421823/1b7e5ea3-e506-45c4-b79d-7f488baf21c9)


That way, all your users will be able to test the admin functionality that you nicely implemented. 👍

Reply:
@Meltrust  Thank you for your helpful and constructive feedback.